apps/services/user-profile/src/app/v2/dto/user-profileDto.ts

```diff
@@ -0,0 +1,56 @@
+import { IsBoolean, IsEnum, IsOptional, IsString } from 'class-validator'
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger'
+
+import { DataStatus } from '../../user-profile/types/dataStatusTypes'
+import { DataType } from 'sequelize-typescript'
+
+export class UserProfileDto {
+  @ApiProperty()
+  @IsString()
+  readonly nationalId: string
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsString()
```

Comment:
`class-validator` contains `IsEmail` decorator that we can use instead to validate email syntax.